### PR TITLE
feat(actions): Support Rego in Kubernetes Action input

### PR DESCRIPTION
**Pull request by simar7**
This PR adds the support for the following:

```
- name: my-k8s
  type: kubernetes
  enable: true
  kube-namespace: "default"                         # Required. Kubernetes namespace to use.
  kube-config-file: "/path/to/kubeconfig"           # Required. Path to .kubeconfig file
  kube-label-selector: "app=nginx-app"              # Required, if specifying labels or annotations.
  kube-actions:
    labels:
      foo-label: "bar-value"                        # Required. Label to add.
      bar-label: event.input.SigMetadata.ID         # Optional. It is also possible to add labels based on event inputs.
    annotations:
      foo-annotation: "bar-value"
      bar-annotation: event.input.SigMetadata.ID
```

Signed-off-by: Simar <simar@linux.com>

### CODE REVIEW

File: outputs/kubernetes.go [hunk 1]
```diff
@@ -53,22 +56,47 @@ func (k *KubernetesClient) Init() error {
 	return nil
 }
 
+func (k KubernetesClient) prepareInputs(input map[string]string) map[string]map[string]string {
+	a := make(map[string]map[string]string)
+
+	for key, m := range k.KubeActions {
+		for id, val := range m {
+			if strings.HasPrefix(val, "event.input") {
```
Comment (AndreyLevchenko) [inline outputs/kubernetes.go:64]: to make it a bit more readable I would suggest to use constant for `"event.input."`.

Comment (AndreyLevchenko) [reply outputs/kubernetes.go:64]: another suggestion is calculate value in set of `if` statements and then set it in one final line like
`a[key] = map[string]string{id: calculated}`. It should make statement smaller

Comment (simar7) [reply outputs/kubernetes.go]: thanks - handled that.

File: outputs/kubernetes.go [hunk 2]
```diff
@@ -53,22 +60,45 @@ func (k *KubernetesClient) Init() error {
...
-		if len(k.KubeActions["labels"]) > 0 {
+		if len(actions["labels"]) > 0 {
 			retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
 				pod, err := k.clientset.CoreV1().Pods(pod.GetNamespace()).Get(ctx, pod.Name, metav1.GetOptions{})
 				if err != nil {
 					return fmt.Errorf("failed to get updated pod for labeling: %s, err: %w", pod.Name, err)
 				}
 
-				labels := updateMap(pod.GetLabels(), k.KubeActions["labels"])
+				labels := updateMap(pod.GetLabels(), actions["labels"])
```
Comment (krol3) [inline outputs/kubernetes.go:101]: in the case of actions, we have the default keys: labels and annotations. Don't be better to use const to refer them?

**Review by AndreyLevchenko — APPROVED** [review]

File: outputs/kubernetes.go (hunk 2 above)
Comment (simar7) [reply outputs/kubernetes.go]: Yeah good point, thanks - I've updated that.

**Pull request merged**